doc/getting-started/getting-started.rst

```diff
@@ -807,7 +662,7 @@ Launch the User VM
       acrn-dm -m $mem_size -s 0:0,hostbridge \
          --mac_seed $mac_seed \
          $logger_setting \
-         -s 9,virtio-blk,/home/acrn/acrn-work/ubuntu-18.04.6-desktop-amd64.iso \
+         -s 9,virtio-blk,/home/acrn/acrn-work/ubuntu-20.04.3-desktop-amd64.iso \
```

Comment:
```suggestion
         -s 9,virtio-blk,/home/acrn/acrn-work/ubuntu-20.04.4-desktop-amd64.iso \
```

Reply:
Actually, we should use the configurator to make this change as we did to tweak the mem_size to 1024.